components/TipsScreen.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components/native';
+
+import backgroundImage from '../assets/sharon-mccutcheon-62vi3TG5EDg-unsplash.jpg';
```

Comment:
Super small thing, but next time when you upload images to your project, try to change their names to something shorter